packages/backend/src/graphql/schema.gql-to-typescript.ts

```diff
@@ -0,0 +1,5 @@
+import ExecutionStep from '@/models/execution-step'
+import TableMetadata from '@/models/table-metadata'
+
+export type TableMetadataGraphQLType = TableMetadata
+export type ExecutionStepGraphQLType = ExecutionStep
```

Comment:
can i check this is required whenever we create custom resolvers? like is it without this, the custom resolver cannot read the type from the `models`?

Reply:
Correct - without this, the custom resolver cannot read our objection models.